openmdao/test_suite/components/paraboloid_distributed.py

```diff
@@ -83,40 +84,30 @@ def initialize(self):
                              desc="Size of input and output vectors.")
 
     def setup(self):
-        arr_size = self.options['arr_size']
-        comm = self.comm
-        rank = comm.rank
 
-        sizes, offsets = evenly_distrib_idxs(comm.size, arr_size)
-        start = offsets[rank]
-        io_size = sizes[rank]
-        self.offset = offsets[rank]
-        end = start + io_size
+        arr_size = self.options['arr_size']
```

Comment:
This will be a bit of a pain, but I think we need to keep the old tests that cover src_indices in add_in/output as they are, and add new ones for these notebook-imported classes with the `distributed=True` way of doing things.  Right now, Bret is fixing bugs in the "new" way, but if he accidentally breaks something in the "old" way and we've removed test coverage for that feature, we'll potentially have a release with 2 broken ways to do distributed vars.